### PR TITLE
EZP-31020: Revised UI labels and messages

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-31020
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes/no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

https://github.com/ezsystems/ezplatform-admin-ui/pull/1115

### CODE REVIEW

File: src/bundle/Resources/views/forgot_password/success.html.twig [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 {%- block content -%}
         {{ 'ezplatform.forgot_password.success'|trans|raw
-        |desc('<p>We have sent an email to the email address of your account. You should receive the email shortly.</p>
+        |desc('<p>We have sent an email to the email address connected with your account. You should receive the email shortly.</p>
```
Comment (michalmilc90) [inline src/bundle/Resources/views/forgot_password/success.html.twig]:
```suggestion
        |desc('<p>We have sent an email to the email address assigned to your account. You should receive the email shortly.</p>
```

File: src/bundle/Resources/views/forgot_password/success.html.twig [hunk 2]
```diff
@@ -4,8 +4,8 @@
...
             <p>If the email is slow to arrive, check your spam folders.</p>
-            <p>If you receive no email, it’s that your username or email is unknown to us, so please double check.</p>
-            <p>If you reset your password multiple times, only the most recent email link will be valid</p>') }}
+            <p>If you receive no email, it means that your username or email is not in the system.</p>
```
Comment (michalmilc90) [inline src/bundle/Resources/views/forgot_password/success.html.twig]:
```suggestion
            <p>If you don't receive an email, it means that your username or email is not in the system.</p>
```

File: src/bundle/Resources/views/forgot_password/with_login.html.twig [hunk 3]
```diff
@@ -5,12 +5,12 @@
...
     <p>
-        {{ 'ezplatform.forgot_password.login'|trans|desc('This email is connected with several accounts. Please enter your login.') }}
+        {{ 'ezplatform.forgot_password.login'|trans|desc('This email is connected with several accounts. Enter your login instead.') }}
     </p>
     {{ form_start(form_forgot_user_password_with_login) }}
     {{ form_widget(form_forgot_user_password_with_login) }}
     {{ form_end(form_forgot_user_password_with_login) }}
     <p>
-    {{ 'ezplatform.forgot_password.contact_administrator'|trans|desc('If you do not remember your login, please contact the site administrator.') }}
+    {{ 'ezplatform.forgot_password.contact_administrator'|trans|desc('If you do not remember your login, contact the site administrator.') }}
```
Comment (michalmilc90) [inline src/bundle/Resources/views/forgot_password/with_login.html.twig]:
```suggestion
    {{ 'ezplatform.forgot_password.contact_administrator'|trans|desc('If you do not remember your login, contact the site Administrator.') }}
```

File: src/bundle/Resources/views/reset_password/success.html.twig [hunk 4]
```diff
@@ -6,6 +6,6 @@
     <h2>{{ 'ezplatform.reset_password.change_password'|trans|desc('Change password') }}</h2>
     <p>
         {{ 'ezplatform.reset_password.success'|trans({ '%login%': path('login') })
-        |desc('Congrats, your password has been changed, you can now <a href="%login%">login</a> to eZ Platform.')|raw }}
+        |desc('Your password has been changed, you can now <a href="%login%">log in</a> to eZ Platform.')|raw }}
```
Comment (michalmilc90) [inline src/bundle/Resources/views/reset_password/success.html.twig]:
```suggestion
        |desc('Your password has been changed. You can now <a href="%login%">log in</a> to eZ Platform.')|raw }}
```

File: src/bundle/Resources/views/forgot_password/success.html.twig [hunk 5]
```diff
@@ -4,8 +4,8 @@
 
 {%- block content -%}
         {{ 'ezplatform.forgot_password.success'|trans|raw
-        |desc('<p>We have sent an email to the email address of your account. You should receive the email shortly.</p>
+        |desc('<p>An email has been sent to the address you provided with instructions on how to reset your password.</p>
             <p>If the email is slow to arrive, check your spam folders.</p>
-            <p>If you receive no email, it’s that your username or email is unknown to us, so please double check.</p>
-            <p>If you reset your password multiple times, only the most recent email link will be valid</p>') }}
+            <p>If you don't receive an email, make sure you've entered the address you registered with, and check your spam folder.</p>
+            <p>If you reset your password multiple times, only the most recent email link will be valid.</p>') }}
```
Comment (supriyabhargava) [inline src/bundle/Resources/views/forgot_password/success.html.twig]:
```suggestion
            <p>If you reset your password multiple times, only the most recent password reset link will be valid.</p>') }}
```

File: src/bundle/Resources/views/forgot_password/with_login.html.twig [hunk 6]
```diff
@@ -5,12 +5,12 @@
...
     <p>
-        {{ 'ezplatform.forgot_password.login'|trans|desc('This email is connected with several accounts. Please enter your login.') }}
+        {{ 'ezplatform.forgot_password.login'|trans|desc('This email is connected with several accounts. Enter your login instead.') }}
     </p>
     {{ form_start(form_forgot_user_password_with_login) }}
     {{ form_widget(form_forgot_user_password_with_login) }}
     {{ form_end(form_forgot_user_password_with_login) }}
     <p>
-    {{ 'ezplatform.forgot_password.contact_administrator'|trans|desc('If you do not remember your login, please contact the site administrator.') }}
+    {{ 'ezplatform.forgot_password.contact_administrator'|trans|desc('If you do not remember your login, contact the site Administrator.') }}
```
Comment (supriyabhargava) [inline src/bundle/Resources/views/forgot_password/with_login.html.twig]:
```suggestion
    {{ 'ezplatform.forgot_password.contact_administrator'|trans|desc('If you do not remember your username, contact the site Administrator.') }}
```

File: src/bundle/Resources/views/register/register_confirmation.html.twig [hunk 7]
```diff
@@ -6,6 +6,6 @@
     <h1>{{ 'ezplatform.registration_confirmation.header'|trans|desc('Your account has been created') }}</h1>
     <p class="user-register-confirmation-message">
         {{ 'ezplatform.registration_confirmation.message'|trans({ '%login%': path('login') })
-        |desc('Thank you for registering an account. You can now <a href="%login%">login</a>')|raw }}    </p>
+        |desc('Thank you for registering an account. You can now <a href="%login%">log in</a>')|raw }}    </p>
```
Comment (supriyabhargava) [inline src/bundle/Resources/views/register/register_confirmation.html.twig]:
```suggestion
        |desc('Thank you for creating an account. You can now <a href="%login%">log in</a>')|raw }}    </p>
```

File: src/bundle/Resources/views/reset_password/success.html.twig [hunk 8]
```diff
@@ -6,6 +6,6 @@
     <h2>{{ 'ezplatform.reset_password.change_password'|trans|desc('Change password') }}</h2>
     <p>
         {{ 'ezplatform.reset_password.success'|trans({ '%login%': path('login') })
-        |desc('Congrats, your password has been changed, you can now <a href="%login%">login</a> to eZ Platform.')|raw }}
+        |desc('Your password has been changed. You can now <a href="%login%">log in</a> to eZ Platform.')|raw }}
```
Comment (supriyabhargava) [inline src/bundle/Resources/views/reset_password/success.html.twig]:
```suggestion
        |desc('Your password has been successfully changed. You can now <a href="%login%">log in</a> to your account.')|raw }}
```

File: src/lib/Form/Type/UserPasswordChangeType.php [hunk 9]
```diff
@@ -23,11 +23,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('oldPassword', PasswordType::class, [
                 'required' => true,
-                'label' => /** @Desc("Old password") */ 'ezplatform.change_user_password.old_password',
+                'label' => /** @Desc("Current password") */ 'ezplatform.change_user_password.old_password',
             ])
             ->add('newPassword', RepeatedType::class, [
                 'type' => PasswordType::class,
-                'invalid_message' => /** @Desc("The password fields must match.") */ 'ezplatform.change_user_password.passwords_must_match',
+                'invalid_message' => /** @Desc("Passwords do not match.") */ 'ezplatform.change_user_password.passwords_must_match',
```
Comment (supriyabhargava) [inline src/lib/Form/Type/UserPasswordChangeType.php]: @inakijv - Wanted to check if we can have a negative 'Do not' in a validation message

File: src/lib/UserSetting/Setting/CharacterCounter.php [hunk 10]
```diff
@@ -114,7 +114,7 @@ private function getTranslatedName(): string
     private function getTranslatedDescription(): string
     {
         return $this->translator->trans(
-            /** @Desc("Displays count of characters typed in Online Editor") */
+            /** @Desc("Displays number of characters typed in Online Editor") */
```
Comment (supriyabhargava) [inline src/lib/UserSetting/Setting/CharacterCounter.php]:
```suggestion
            /** @Desc("Enable character count") */
```

File: src/lib/Form/Type/UserSettings/DateTimeFormatType.php [hunk 11]
```diff
@@ -21,14 +21,14 @@ class DateTimeFormatType extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options): void
     {
         $builder->add('date_format', ChoiceType::class, [
-            'label' => /** @Desc("Format for displaying Date") */ 'ezplatform.date_time_format.date_format.label',
+            'label' => /** @Desc("Format for displaying date") */ 'ezplatform.date_time_format.date_format.label',
```
Comment (supriyabhargava) [inline src/lib/Form/Type/UserSettings/DateTimeFormatType.php]:
```suggestion
            'label' => /** @Desc("Date format") */ 'ezplatform.date_time_format.date_format.label',
```

File: src/lib/Form/Type/UserSettings/DateTimeFormatType.php [hunk 12]
```diff
@@ -21,14 +21,14 @@ class DateTimeFormatType extends AbstractType
...
             'choices' => $options['date_format_choices'],
             'multiple' => false,
             'required' => true,
         ]);
 
         $builder->add('time_format', ChoiceType::class, [
-            'label' => /** @Desc("Format for displaying Time") */ 'ezplatform.date_time_format.time_format.label',
+            'label' => /** @Desc("Format for displaying time") */ 'ezplatform.date_time_format.time_format.label',
```
Comment (supriyabhargava) [inline src/lib/Form/Type/UserSettings/DateTimeFormatType.php]:
```suggestion
            'label' => /** @Desc("Time format") */ 'ezplatform.date_time_format.time_format.label',
```

File: src/lib/UserSetting/Setting/FullDateTimeFormat.php [hunk 13]
```diff
@@ -81,7 +81,7 @@ public function mapFieldForm(
     protected function getTranslatedName(): string
     {
         return $this->translator->trans(
-            /** @Desc("Long Date & Time format") */
+            /** @Desc("Long date and time format") */
```
Comment (supriyabhargava) [inline src/lib/UserSetting/Setting/FullDateTimeFormat.php]:
```suggestion
            /** @Desc("Long Date and Time format") */
```
Date and Time are always capitalized.

File: src/lib/UserSetting/Setting/FullDateTimeFormat.php [hunk 14]
```diff
@@ -94,7 +94,7 @@ protected function getTranslatedName(): string
     protected function getTranslatedDescription(): string
     {
         return $this->translator->trans(
-            /** @Desc("Format for displaying Date & Time") */
+            /** @Desc("Format for displaying date and time") */
```
Comment (supriyabhargava) [inline src/lib/UserSetting/Setting/FullDateTimeFormat.php]:
```suggestion
            /** @Desc("Date and Time format") */
```

File: src/lib/UserSetting/Setting/Language.php [hunk 15]
```diff
@@ -111,7 +111,7 @@ private function getTranslatedName(): string
     private function getTranslatedDescription(): string
     {
         return $this->translator->trans(
-            /** @Desc("The language of the administration panel") */
+            /** @Desc("The language of the Back Office") */
```
Comment (supriyabhargava) [inline src/lib/UserSetting/Setting/Language.php]:
```suggestion
            /** @Desc("Back Office language") */
```

File: src/lib/UserSetting/Setting/ShortDateTimeFormat.php [hunk 16]
```diff
@@ -80,7 +80,7 @@ public function mapFieldForm(
     protected function getTranslatedName(): string
     {
         return $this->translator->trans(
-            /** @Desc("Short Date & Time format") */
+            /** @Desc("Short date and time format") */
```
Comment (supriyabhargava) [inline src/lib/UserSetting/Setting/ShortDateTimeFormat.php]:
```suggestion
            /** @Desc("Short Date and Time format") */
```

File: src/lib/UserSetting/Setting/ShortDateTimeFormat.php [hunk 17]
```diff
@@ -93,7 +93,7 @@ protected function getTranslatedName(): string
     protected function getTranslatedDescription(): string
     {
         return $this->translator->trans(
-            /** @Desc("Format for displaying Date & Time") */
+            /** @Desc("Format for displaying date and time") */
```
Comment (supriyabhargava) [inline src/lib/UserSetting/Setting/ShortDateTimeFormat.php]:
```suggestion
            /** @Desc("Date and Time format") */
```

File: src/lib/UserSetting/Setting/Timezone.php [hunk 18]
```diff
@@ -94,7 +94,7 @@ private function getTranslatedName(): string
     private function getTranslatedDescription(): string
     {
         return $this->translator->trans(
-            /** @Desc("Time Zone in use for displaying Date & Time") */
+            /** @Desc("Time Zone in use for displaying date and time") */
```
Comment (supriyabhargava) [inline src/lib/UserSetting/Setting/Timezone.php]:
```suggestion
            /** @Desc("User Time Zone") */
```

File: src/lib/Validator/Constraints/UserPassword.php [hunk 19]
```diff
@@ -26,7 +26,7 @@ public static function getTranslationMessages(): array
     {
         return [
             Message::create('ezplatform.change_user_password.not_match', 'validators')
-                ->setDesc('Old password does not match.'),
+                ->setDesc('Incorrect current password.'),
```
Comment (supriyabhargava) [inline src/lib/Validator/Constraints/UserPassword.php]:
```suggestion
                ->setDesc('Current password is incorrect.'),
```

File: src/lib/UserSetting/Setting/FullDateTimeFormat.php (hunk 13 above)
Comment (DominikaK) [reply src/lib/UserSetting/Setting/FullDateTimeFormat.php]: Discussed face-to-face.

**Pull request merged**